tests/integration/orchestration/prefect/flows/test_sftp.py

```diff
@@ -0,0 +1,16 @@
+"""'test_sftp.py'."""
+
+from viadot.orchestration.prefect.flows import sftp_to_adls
+
+
+def test_sftp_to_adls(azure_key_vault_secret, adls_azure_key_vault_secret):
+    """Test SFTP server prefect flow."""
+    state = sftp_to_adls(
+        azure_key_vault_secret=azure_key_vault_secret,
+        file_name="Data Lake- V-F Products with Prices.tsv",
```

Comment:
Do we need this test here?

Reply:
I don't, I explained on the viadot 2 training that we only need to focus on unit tests, since orchestration should be a thin layer on top, with no additional logic, so not much to test there.

Reply:
Removed. PR is ready. 
There is only an error about the Private Key in the code, but everything there is checked and not sensitive.